docs/config.ini

```diff
@@ -1,13 +1,13 @@
 # below are the server-wide settings for documentation
 [help_settings]
-url_base = http://edx.readthedocs.org/projects/edx-partner-course-staff
-version = latest
+url_base = http://edx.readthedocs.org/projects/open-edx-ca
+version = dogwood
```

Comment:
didn't we have to change the version when we rebuilt, to named-release-dogwood.rc ?